app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+# Service to download ftp files from the serve
+class User < ApplicationRecord
+  has_many :posts, dependent: :destroy
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  def recent_posts
+    @posts = Post.last(3)
+  end
```

Comment:
- [ ] This method, as implemented also returns posts from other users. Remember that every user already has the relationship of `has_many: posts`. So, if we wanted to return the three recent posts for a particular user, we only need to refer to the relationship property which is `posts`, and not create a new variable i.e. `@posts`. 

  Looking at the screenshot below shows that querying for the first user's posts also fetches posts created by the second user, be keen to check on the `user_id` field. So, the logic is not right. Please let's improve on this:  
  
  ![image](https://user-images.githubusercontent.com/45002654/151278695-074981e8-b2a5-4d76-9d8e-e7ec0270f69e.png)


  SUGGESTION
  ```ruby
    def recent_posts
      posts.order('created_at Desc').limit(3)
    end
  ```